### PR TITLE
Updated staticrange IDL file

### PR DESCRIPTION
**Pull request by lukebjerring**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-06 22:24:30
Finished: 2018-03-06 22:32:40


<h2>Failing Jobs</h2>
<ul>

<li>chrome:dev</li>

<li>safari:11.0</li>

<li>MicrosoftEdge:16.16299</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25397)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/350037874)

**Comment by foolip** [comment]
Looks like there aren't any tests for staticrange at all yet. Can you add a staticrange/interfaces.html?

**Comment by foolip** [comment]
> Looks like there aren't any tests for staticrange at all yet. Can you add a staticrange/interfaces.html?

Still the case.

**Comment by lukebjerring** [comment]
PTAL

File: staticrange/idlharness.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<title>Gamepad IDL tests</title>
```
Comment (foolip) [inline staticrange/idlharness.html]: Gamepad copypasta

**Review by foolip — APPROVED** [review]
LGTM % copypasta

**Pull request merged**